bindings/python/pydeck/docs/data_utils.md

```diff
@@ -0,0 +1,43 @@
+pydeck.data\_utils
+=======
+
+## pydeck.data\_utils.autocompute\_viewport
+
+```python
+def autocompute_viewport(
+    points,
+    view_proportion=1,
+    viewport_type=pydeck.ViewState)
+```
+
+Automatically computes a zoom level for the points passed in.
```

Comment:
Nits:
- "Computes a view state (long/lat and zoom level)"?
- Is the `auto` in `autocompute` justified, seeing as this function is explicitly and not automatically called?